modin/_compat/pandas_api/latest/base.py

```diff
@@ -239,6 +306,7 @@ def rolling(
             on=on,
             axis=axis,
             closed=closed,
+            step=step,
```

Comment:
It seems to me that `self._rolling` doesn't handle `step` parameter correctly. Could you double check it?